content/intro-to-storybook/vue/pt/conclusion.md

```diff
@@ -3,20 +3,20 @@ title: 'Conclusão'
 description: 'Fusão de todo o conhecimento adquirido e aprendizagem de técnicas extra de Storybook'
 ---
 
-Parabéns! Você criou sua primeira interface do usuário no Storybook. Ao longo do caminho, você aprendeu a criar, compor, testar e implantar componentes da interface do usuário. Se você seguiu o tutorial, seu repositório e o Storybook implantado devem ficar assim:
+Parabéns! Acabou de criar o primeiro interface de utilizador com o Storybook. Ao longo do caminho, aprendeu a criar, compor, testar e implementar componentes de interface de utilizador. Se seguiu o tutorial o repositório e o Storybook deverão ficar assim:
 
 [📕 **GitHub repo: chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code)
 <br/>
 [🌎 **Storybook implementado**](https://clever-banach-415c03.netlify.com/)
 
-O Storybook é uma ferramenta poderosa para React, Vue e Angular.
-Possui uma comunidade de desenvolvedores próspera e uma grande variedade de complementos. Esta introdução arranha a superfície do que é possível. Estamos confiantes de que, ao adotar o Storybook, você ficará impressionado com a produtividade da criação de UIs duráveis.
+O Storybook é uma ferramenta bastante poderosa para React, Vue e Angular.
+Possui uma comunidade de programadores próspera e uma grande variedade de extras. Esta introdução arranha a superfície do que é possível fazer. Estamos confiantes que ao adotar o Storybook, ficará impressionado com o quão produtivo é construir IUs duradouros.
 
 ## Saber mais
 
-Pretende saber mais? Aqui ficam algumas dicas que irão ajudar.
+Pretende saber mais? Aqui ficam algumas recursos que irão ajudar.
```

Comment:
If you don't mind, change the genre to masculin to avoid introducing a grammar error. It should be "alguns recursos", not "algumas recursos". And this should be good to merge